x/gov/keeper.go

```diff
@@ -128,28 +133,37 @@ func (keeper Keeper) activateVotingPeriod(ctx sdk.Context, proposal Proposal) {
 // =====================================================
 // Procedures
 
-// Gets procedure from store. TODO: move to global param store and allow for updating of this
-func (keeper Keeper) GetDepositProcedure() DepositProcedure {
-	return DepositProcedure{
-		MinDeposit:       sdk.Coins{sdk.NewCoin("steak", 10)},
-		MaxDepositPeriod: 200,
-	}
+// Returns the current Deposit Procedure from the global param store
+func (keeper Keeper) GetDepositProcedure(ctx sdk.Context) DepositProcedure {
+	var depositProcedure DepositProcedure
+	keeper.ps.Get(ctx, "gov/depositprocedure", &depositProcedure)
+	return depositProcedure
 }
 
-// Gets procedure from store. TODO: move to global param store and allow for updating of this
-func (keeper Keeper) GetVotingProcedure() VotingProcedure {
-	return VotingProcedure{
-		VotingPeriod: 200,
-	}
+// Returns the current Voting Procedure from the global param store
+func (keeper Keeper) GetVotingProcedure(ctx sdk.Context) VotingProcedure {
+	var votingProcedure VotingProcedure
+	keeper.ps.Get(ctx, "gov/votingprocedure", &votingProcedure)
+	return votingProcedure
 }
 
-// Gets procedure from store. TODO: move to global param store and allow for updating of this
-func (keeper Keeper) GetTallyingProcedure() TallyingProcedure {
-	return TallyingProcedure{
-		Threshold:         sdk.NewRat(1, 2),
-		Veto:              sdk.NewRat(1, 3),
-		GovernancePenalty: sdk.NewRat(1, 100),
-	}
+// Returns the current Tallying Procedure from the global param store
+func (keeper Keeper) GetTallyingProcedure(ctx sdk.Context) TallyingProcedure {
+	var tallyingProcedure TallyingProcedure
+	keeper.ps.Get(ctx, "gov/tallyingprocedure", &tallyingProcedure)
+	return tallyingProcedure
+}
+
+func (keeper Keeper) setDepositProcedure(ctx sdk.Context, depositProcedure DepositProcedure) {
+	keeper.ps.Set(ctx, "gov/depositprocedure", &depositProcedure)
```

Comment:
these strings should be well defined constants at top of file.

Also might be nicer as `gov/procedure/deposit`

Reply:
Addressed.